apps/system/js/accessibility.js

```diff
@@ -363,6 +394,18 @@
     },
 
     /**
+     * Check for Hints speech/timer and clear.
+     * @memberof Accessibility.prototype
+     */
+    checkHintsStatus: function ar_checkHintsStatus() {
```

Comment:
I was just thinking that the name is slightly misleading. I think it should be clearHints instead of checkHintsStatus, since we mainly doing that.